src/main/java/edu/greenblitz/tobyDetermined/subsystems/RotatingBelly/RotatingBelly.java

```diff
@@ -1,30 +1,26 @@
 package edu.greenblitz.tobyDetermined.subsystems.RotatingBelly;
 
-import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
-import com.ctre.phoenix.motorcontrol.can.TalonSRX;
-import edu.greenblitz.tobyDetermined.RobotMap;
 import edu.greenblitz.tobyDetermined.subsystems.GBSubsystem;
-import edu.wpi.first.wpilibj.DigitalInput;
+import edu.greenblitz.tobyDetermined.subsystems.RotatingBelly.IO.RotatingBellyIOTalonSRX;
+import edu.greenblitz.tobyDetermined.subsystems.RotatingBelly.IO.RotatingBellyInputsAutoLogged;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class RotatingBelly extends GBSubsystem {
 
 
     private static RotatingBelly instance;
-    private TalonSRX motor;
-    private DigitalInput limitSwitch;
+
+    private RotatingBellyIOTalonSRX io;
```

Comment:
What is the purpose of having a genric rotating belly if we force using one of the types?